src/pragha-musicobject-mgmt.c

```diff
@@ -44,16 +43,9 @@ new_musicobject_from_file(const gchar *file)
 
 	g_free (mime_type);
 
-	ret = pragha_musicobject_set_tags_from_file (mobj, file);
+	pragha_musicobject_set_tags_from_file (mobj, file);
 
```

Comment:
Hi @wymiatacz 
This is the only change that can worry.. 

We are opening files that can not read taglib .. Ok .. I do not like, but alreally what we are already doing ..

It's something that always makes me noise .. I'm just saying I want to rethink it well ...  :pensive:

You must check as files are added from the command line, with MPRIS, etc .. :wink: